src/taskOperations.js

```diff
@@ -0,0 +1,147 @@
+            }
+          });
+        };
+      };
+    });
+  }
+
+  // static completeTask = (index) => {
+  //   TaskOperations.tasks[index].completed = true;
+  //   localStorage.setItem('datas', JSON.stringify(TaskOperations.tasks));
+  // }
+
+  // static uncompleteTask = (index) => {
+  //   TaskOperations.tasks[index].completed = false;
+  //   localStorage.setItem('datas', JSON.stringify(TaskOperations.tasks));
+  // }
+
+  // static sortTask = () => {
+  //   TaskOperations.tasks.sort((a, b) => a.index - b.index);
+  //   localStorage.setItem('datas', JSON.stringify(TaskOperations.tasks));
+  // }
+
+  static renderTasks = (task) => {
+    const taskHolder = document.createElement('li');
+    taskHolder.setAttribute('id', task.index);
+    taskHolder.setAttribute('class', 'taskItem');
+    taskHolder.innerHTML = `
+      <span><input type="checkbox" class="checkbox" ${task.completed ? 'checked' : 'unchecked'}>
+      <span class="editDescription">${task.description}</span></span><span class="btnHolder"><i class="fa-solid fa-trash-can"></i><span class="fa-solid fa-ellipsis-vertical fa-lg"></span></span>`;
+    TaskOperations.taskList.appendChild(taskHolder);
+
+    const deleteBtn = document.querySelector('#taskList li:last-child i');
+    deleteBtn.addEventListener('click', () => {
+      TaskOperations.deleteTask(task.index);
+    });
+
+    const editProperty = document.querySelector('#taskList li:last-child span .editDescription');
+    editProperty.addEventListener('click', () => {
+      TaskOperations.editTask(task.index);
+    });
+  }
+
+  static updateTasks = () => {
+    if (localStorage.getItem('datas')) {
+      TaskOperations.tasks = JSON.parse(localStorage.getItem('datas'));
+      TaskOperations.tasks.forEach((task) => {
+        TaskOperations.renderTasks(task);
+      });
+    } else {
+      TaskOperations.tasks = [];
+    }
+  }
+
+  static init = () => {
+    TaskOperations.loadTasks();
+    TaskOperations.renderTasks();
+  }
+
+  // static addEventListeners = () => {
+  //   const addBtn = document.querySelector('.fa-plus');
+  //   const taskList = document.querySelector('#taskList');
+
+  //   addBtn.addEventListener('click', TaskOperations.addTask);
+  //   taskList.addEventListener('click', (e) => {
+  //     if (e.target.classList.contains('fa-ellipsis-vertical')) {
+  //       const index = e.target.parentElement.parentElement.id;
+  //       if (e.target.parentElement.parentElement.classList.contains('taskItem')) {
+  //         TaskOperations.deleteTask(index);
+  //         TaskOperations.renderTasks();
+  //       } else if (e.target.parentElement.parentElement.classList.contains('checkbox')) {
+  //         if (e.target.parentElement.parentElement.checked) {
+  //           TaskOperations.completeTask(index);
+  //         } else {
+  //           TaskOperations.uncompleteTask(index);
+  //         }
+  //         TaskOperations.renderTasks();
+  //       }
+  //     }
+  //   });
+  // }
```

Comment:
- [ ] As I mentioned before, please try to avoid commented code, so please remove it from your codebase.